hazelcast/src/main/java/com/hazelcast/internal/serialization/SerializationServiceBuilder.java

```diff
@@ -63,5 +64,7 @@
 
     SerializationServiceBuilder setInitialOutputBufferSize(int initialOutputBufferSize);
 
+    SerializationServiceBuilder setClusterService(ClusterService clusterService);
```

Comment:
I understand this. But I feel hesitant introducing a dependency on a big object like ClusterService for the sake of obtaining a version. Of course it is the easiest thing to do in the short term, but it increases coupling between the 2 services.